lib/vc.js

```diff
@@ -353,18 +353,21 @@ async function _verifyPresentation(options = {}) {
   const documentLoader = options.documentLoader || defaultDocumentLoader;
 
   // if verifiableCredentials are present, verify them, individually
-  let vcResult;
+  let credentialResults;
   const credentials = jsonld.getValues(presentation, 'verifiableCredential');
   if(credentials.length > 0) {
     // verify every credential in `verifiableCredential`
-    vcResult = await Promise.all(credentials.map(credential => {
+    credentialResults = await Promise.all(credentials.map(credential => {
       return verifyCredential({credential, documentLoader, ...options});
     }));
 
-    const allCredentialsVerified = vcResult.every(r => r.verified);
+    const allCredentialsVerified = credentialResults.every(r => r.verified);
 
     if(!allCredentialsVerified) {
-      return {verified: false, error: vcResult.map(r => r.error).flat()};
+      return {
+        error: credentialResults.map(r => r.error).flat(),
```

Comment:
Array.flat() is not supported in Node 10.x https://github.com/digitalbazaar/vc-js/issues/57.

Reply:
Addressed by: 2394f2c